audbackend/core/api.py

```diff
@@ -15,6 +15,31 @@
 r"""Backend registry."""
 
 
+def available() -> typing.Dict[str, typing.List[Backend]]:
+    r"""List available backends.
+
+    Returns:
+        sorted dictionary with backends
+
+    Examples:
+        >>> list(available())
+        ['artifactory', 'file-system']
+        >>> available()['artifactory']
+        [('Artifactory', 'https://audeering.jfrog.io/artifactory', 'repo')]
```

Comment:
Maybe we should change the output:

```
[('audbackend.Artifactory', 'https://audeering.jfrog.io/artifactory', 'repo')]
```

?

Reply:
You are right, we either stay with only the class name or we expand to the full module name, i.e.:

```
[('audbackend.core.artifactory.Artifactory', 'https://audeering.jfrog.io/artifactory', 'repo')]
```

Reply:
Since we may have two implementations with the same class name in two different modules, I decided to change to the full name. We now have:

![image](https://user-images.githubusercontent.com/10383417/206717841-1860035b-9de3-44f6-a43c-614331c05b6e.png)

and 

![image](https://user-images.githubusercontent.com/10383417/206717878-4b72a55c-ff91-4eb8-94f7-bd78c3f3abfd.png)